src/app/components/LinkButton/LinkButton.tsx

```diff
@@ -0,0 +1,57 @@
+import Link, { LinkProps } from 'next/link';
+
+interface LinkButtonProps extends LinkProps {
+  children: React.ReactNode;
+  variant: 'primary' | 'secondary';
+  disabled?: boolean;
+}
+export function LinkButton({
+  children,
+  variant,
+  href,
+  disabled = false,
+  ...props
+}: LinkButtonProps) {
+  if (variant === 'secondary') {
+    return (
+      <Link
+        href={href}
+        className="usa-button hover:bg-red items-center justify-start gap-2.5 rounded border-2 border-[#224a58] bg-white px-5 py-3 hover:border-2"
```

Comment:
Is there a way to tell in Figma what the hover state of the button should look like? I put very obvious placeholders (like the `hover:bg-red` here) that I figured we could update later. If we already have this info somewhere, let me know and I can make the update!

Reply:
I don't think there is. The way i've seen designers do it is they specifically have a section with all the button states. I don't see that on figma for this